app/backend/src/couchers/constants.py

```diff
@@ -44,3 +44,11 @@
 ACTIVENESS_PROBE_TIME_REMINDERS = [timedelta(days=0), timedelta(days=4, hours=8)]
 # total time from initiation after which to expire the probe
 ACTIVENESS_PROBE_EXPIRY_TIME = timedelta(days=14)
+
+# request quotas
+HOST_REQUEST_DAILY_WARNING_QUOTA = 20
+HOST_REQUEST_DAILY_BLOCKING_QUOTA = 80
```

Comment:
Don't use the word `blocking` since the user is not blocked, call it a `limit`, or hard limit, whatever

Reply:
Adjusted the wording everywhere now!